app/views/edit_phone/code.html.erb

```diff
@@ -1,4 +1,4 @@
-<% content_for :title, t("mfa.phone.code.change_heading") %>
+<% content_for :title, t("mfa.phone.code.page_title") %>
 <%= render "govuk_publishing_components/components/heading", {
   text: yield(:title),
```

Comment:
On this page, the `h1` takes its content from the `:title` which means that the `h1` will change to **Check your phone** whereas it used to be **Enter your security code**, is that what we want?

Reply:
Yes, I think so. This is the page: http://www.login.service.dev.gov.uk/account/edit/phone/code

It was 'Enter your security code' but that was also the label on the only form element on that page, so now it makes more sense (I think) and title and page title match.

![Screenshot 2021-03-02 at 15 06 36](https://user-images.githubusercontent.com/861310/109668300-f27c9c00-7b68-11eb-8da6-368bd0bfabf6.png)